Detections/AWSCloudTrail/AWS_ChangeToRDSDatabase.yaml

```diff
@@ -24,4 +26,4 @@ query: |
       or EventName == "CreateDBSecurityGroup"
       or EventName == "DeleteDBSecurityGroup"
       or EventName == "RevokeDBSecurityGroupIngress"
-  | project TimeGenerated, EventName, EventTypeName, UserIdentityAccountId, UserIdentityPrincipalid, UserAgent, UserIdentityUserName, SessionMfaAuthenticated, SourceIpAddress, AWSRegion, EventSource, AdditionalEventData, ResponseElements
\ No newline at end of file
+  | project TimeGenerated, EventName, EventTypeName, UserIdentityAccountId, UserIdentityPrincipalid, UserAgent, UserIdentityUserName, SessionMfaAuthenticated, SourceIpAddress, AWSRegion, EventSource, AdditionalEventData, ResponseElements, AccountCustomEntity = UserIdentityUserName, IPCustomEntity = SourceIpAddress
```

Comment:
suggest we consistently put the custom entity bit on separate line as:
| extend AccountCustomEntity = UserIdentityUserName, IPCustomEntity = SourceIpAddress
This means we maintain the initial columns as intended by the query, and we have a common approach across all queries of doing this tagging in the last line.